draft-ietf-quic-recovery.md

```diff
@@ -405,10 +410,11 @@ On subsequent RTT samples, smoothed_rtt and rttvar evolve as follows:
 
 ~~~
 ack_delay = min(Ack Delay in ACK Frame, max_ack_delay)
+corrected_rtt = latest_rtt
 if (min_rtt + ack_delay < latest_rtt):
-  latest_rtt = latest_rtt - ack_delay
-smoothed_rtt = 7/8 * smoothed_rtt + 1/8 * latest_rtt
-rttvar_sample = abs(smoothed_rtt - latest_rtt)
+  corrected_rtt = latest_rtt - ack_delay
```

Comment:
Maybe `adjusted_rtt` instead.  "Corrected" implies that there is some notion of "correctness" that applies.